scheduling-api-http/build.gradle

```diff
@@ -57,11 +57,23 @@ dependencies {
     // in order to start the application from your IDE
     providedRuntime 'org.springframework.boot:spring-boot-starter-tomcat'
 
-    compile 'org.hibernate:hibernate-entitymanager'
-    compile 'org.hibernate:hibernate-core'
+    compile 'org.hibernate:hibernate-entitymanager:5.1.16.Final'
+    compile 'org.hibernate:hibernate-core:5.1.16.Final'
 
     runtime 'org.hsqldb:hsqldb'
 
+    //support for Oracle DB version 12.1.0.2
```

Comment:
can you put a comment to explain why it is commented out, and how it can be used (credentials)?